sycl/test/basic_tests/sampler/sampler.cpp

```diff
@@ -50,37 +48,10 @@ int main() {
          A.get_filtering_mode() == B.get_filtering_mode());
 
   // Check assignment operator
-  if (!Queue.is_host()) {
-    // OpenCL sampler
-    cl_int Err = CL_SUCCESS;
-#ifdef CL_VERSION_2_0
-    const cl_sampler_properties sprops[] = {
-        CL_SAMPLER_NORMALIZED_COORDS,
-        static_cast<cl_sampler_properties>(true),
-        CL_SAMPLER_ADDRESSING_MODE,
-        static_cast<cl_sampler_properties>(CL_ADDRESS_REPEAT),
-        CL_SAMPLER_FILTER_MODE,
-        static_cast<cl_sampler_properties>(CL_FILTER_LINEAR),
-        0};
-    cl_sampler ClSampler =
-        clCreateSamplerWithProperties(Queue.get_context().get(), sprops, &Err);
-#else
-    cl_sampler ClSampler =
-        clCreateSampler(Queue.get_context().get(), true, CL_ADDRESS_REPEAT,
-                        CL_FILTER_LINEAR, &Err);
-#endif
-    // If device doesn't support sampler - skip it
-    if (Err == CL_INVALID_OPERATION)
-      return 0;
+  B = sycl::sampler(sycl::coordinate_normalization_mode::normalized,
+                    sycl::addressing_mode::repeat,
+                    sycl::filtering_mode::linear);
```

Comment:
It look like this test in failing on non-CUDA back-ends.
@smaslov-intel, @romanovvlad, FYI.

> die: piextKernelSetArgSampler: not implemented

@StuartDAdams, I suggest marking this test as expected to fail on non-CUDA back-ends until we implement `piextKernelSetArgSampler` there.

Reply:
At least LevelZero plug-in should be updated.

Reply:
Just saw this message, I've just pushed a new commit that updates the test with "XFAIL: level0".